src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.BasicAuth.cs

```diff
@@ -0,0 +1,114 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using System.Net.Test.Common;
+using System.Text;
+using System.Threading.Tasks;
+
+using Microsoft.DotNet.XUnitExtensions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Http.Functional.Tests
+{
+    [ConditionalClass(typeof(PlatformDetection), nameof(PlatformDetection.IsNotBrowser))]
+    public class HttpClientHandlerTest_BasicAuth : HttpClientHandlerTestBase
+    {
+        public HttpClientHandlerTest_BasicAuth(ITestOutputHelper output) : base(output)
```

Comment:
nit: I think we do not need constructor if we are not storing `ITestOutputHelper` for later use.

Reply:
`HttpClientHandlerTestBase` does not have a parameterless constructor.